gomint-api/src/main/java/io/gomint/world/block/BlockSign.java

```diff
@@ -8,12 +8,11 @@
 package io.gomint.world.block;
 
 import io.gomint.world.block.data.LogType;
-import io.gomint.world.block.data.SignDirection;
 
 import java.util.List;
 
 /**
- * @author geNAZt
+ * @author derklaro
```

Comment:
```suggestion
 * @author geNAZt
 * @author derklaro
```
or just leave author stuff to the commit history